src/core/Stats.tsx

```diff
@@ -16,7 +16,7 @@ export function Stats({ showPanel = 0, className, parent }: Props): null {
       const node = (parent && parent.current) || document.body
       stats.showPanel(showPanel)
       node?.appendChild(stats.dom)
-      if (className) stats.dom.classList.add(className)
+      if (className) stats.dom.classList.add(...className.split(' ').filter((cls) => cls))
```

Comment:
The `.filter((cls) => cls)` part is useful to remove empty strings from the array.

For example:
```js
const className = ' foo bar';
const classListWithEmptyStrings = className.split(' ');            // [' ', 'foo', 'bar']
const classList = classListWithEmptyStrings.filter((cls) => cls); // ['foo', 'bar']
```